stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -95,17 +117,21 @@ class FPDF:
     images: dict[str, _Image]
     annots: defaultdict[int, list[Annotation]]
     links: dict[int, DestinationXYZ]
+    embedded_files: list[Incomplete]
+    embedded_files_per_pdf_ref: dict[Incomplete, Incomplete]
     in_footer: int
     lasth: int
     current_font: _Font
     font_family: str
     font_style: str
+    font_stretching: float
+    char_spacing: int
```

Comment:
`set_char_spacing()` does `self._out(f"BT {spacing:.2f} Tc ET")` where `spacing` is a value of `self.char_spacing`, and the docstring says `float`.

```suggestion
    char_spacing: float
```